sdk/tests/tutorials/ibor/test_orders.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import uuid
+import lusid
+import lusid.models as models
+from lusid import *
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+
+class Orders(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = TestDataUtilities.api_client()
+        cls.orders_api = lusid.OrdersApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+    def get_guid(self):
+        # creates random alphanumeric code
+        return str(uuid.uuid4())[:12]
+
+    @lusid_feature("F4")
+    def test_upsert_simple_order(self):
+        """Test makes a request for a single order."""
+
+        test_scope = "Orders-SimpleUpsert-TestScope"
+        # Create unique order id
+        order = self.get_guid()
+        # Create ResourceId for order
+
+        # Construct order arguments
+        instrument_identifiers = {TestDataUtilities.lusid_luid_identifier: self.instrument_ids[0]}
+        order_id = models.ResourceId(test_scope, order)
+        portfolio_id = ResourceId(test_scope, "OrdersTestPortfolio")
+        # The key:This takes the format {domain}/{scope}/{code}, value: Lusid property value'
```

Comment:
I'm not sure this comment is useful (for a reader), its duplicating the actual api docs